lesson_13/maps_ts/src/lesson13.ts

```diff
@@ -1,7 +1,20 @@
-/**
- * Provide the solution to LeetCode 3146 here:
- * https://leetcode.com/problems/permutation-difference-between-two-strings
- */
 export function findPermutationDifference(s: string, t: string): number {
-  return 0;
-}
+  const stringS = new Map<string, number>();
+  const stringT = new Map<string, number>();
+  let diffSum = 0;
+  for (let i = 0; i < t.length; i++) {
+    stringS.set(s.charAt(i), i);
+    stringT.set(t.charAt(i), i);
+  }
+  for (const key of stringS.keys()) {
+    if (stringT.has(key)) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Shouldn't need to disable the assertions, please remove.

Reply:
I'll find a solution right away. I was having trouble with it yesterday and was causing an error. It was a quick fix that VS code suggested for those lines. Do you have any suggestions on how to fix it or do you think just removing the comments will work still?

Reply:
I made changes thinking that the build ran with removing those lines but it never actually pushed those changes correctly. The typescript file now has an error.